src/Conf.ml

```diff
@@ -1489,8 +1489,6 @@ let validate () =
     `Error (false, "Cannot specify stdin together with other inputs")
   else if has_stdin && Option.is_none !name then
     `Error (false, "Must specify name when reading from stdin")
-  else if !inplace && Option.is_some !name then
```

Comment:
Should we fail if `--name` is used with multiple (more than one) inputs instead ?

Reply:
I don't know. In the case of the use in merge-fmt, currently ocamlformat is called three times on one file each time, always using the same name. In this case, a single call to ocamlformat with one name and all three inputs could be made instead. I don't know how over-specific that seems to you, but there is at least one use case.